### PR TITLE
Fix Sync and Sync Workflow

### PR DESCRIPTION
**Pull request by Imipenem**
**Associated Template/Command/Core**

`cookietemple sync`

**PR Checklist**

 - [x] This comment contains a description of changes (with reason)
 - [x] Referenced issue is linked
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [x] Documentation in `docs` is updated
 - [x] `CHANGELOG.rst` is updated

**Description of changes**

The cookietemple sync Workflow for automatic syncing is now working without modifying the project structure by removing the WF from TEMPLATE branch and double pushing master/development branch

### CODE REVIEW

File: cookietemple/create/templates/available_templates.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ cli:
     python:
         name: Python Commandline Package
         handle: cli-python
-        version: 1.0.0
+        version: 2.0.0
```
Comment (Zethson) [inline cookietemple/create/templates/available_templates.yml:14]: u sure

File: cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/README.rst [hunk 2]
```diff
@@ -40,7 +40,7 @@ Features
 
 Credits
 -------
-
+Updated for sync
```
Comment (Zethson) [inline cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/README.rst:43]: m8

File: cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/testsync.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+Hello, I'm a testfile, that should be included in latest project sync!
```
Comment (Zethson) [inline cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/testsync.txt:1]: I M GONNA INCLUDE U AS WELL NOW

File: cookietemple/create/templates/available_templates.yml (hunk 1 above)
Comment (Imipenem) [reply cookietemple/create/templates/available_templates.yml:14]: WIP, will change this, right now I'm working on this.

File: cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/.github/workflows/sync_project.yml [hunk 4]
```diff
@@ -1,9 +1,6 @@
 name: cookietemple sync
 
-on:
-  schedule:
-    - cron: '0 1 * * *' # 1 am UTC
-  workflow_dispatch:
+on: [push]
```
Comment (Zethson) [inline cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/.github/workflows/sync_project.yml:3]: NO

File: cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/.github/workflows/sync_project.yml [hunk 5]
```diff
@@ -14,12 +11,13 @@ jobs:
           with:
             python-version: 3.8
 
-        - name: Install cookietemple
-          run: pip install cookietemple
+        - name: Clone and install cookietemple
+          run: git clone https://github.com/cookiejar/cookietemple && cd cookietemple && git checkout fix/sync && pip install .
```
Comment (Zethson) [inline cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/.github/workflows/sync_project.yml:15]: NO

File: cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/README.rst (hunk 2 above)
Comment (Imipenem) [reply cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/README.rst:43]: See above

File: cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/testsync.txt (hunk 3 above)
Comment (Imipenem) [reply cookietemple/create/templates/cli/cli_python/{{ cookiecutter.project_slug_no_hyphen }}/testsync.txt:1]: Also see above

File: cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/.github/workflows/sync_project.yml (hunk 4 above)
Comment (Imipenem) [reply cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/.github/workflows/sync_project.yml:3]: See above

File: cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/.github/workflows/sync_project.yml (hunk 5 above)
Comment (Imipenem) [reply cookietemple/create/templates/common_files/{{cookiecutter.commonName}}/.github/workflows/sync_project.yml:15]: See above

**Review by Zethson — APPROVED** [review]
LGTM

**Pull request merged**